### DISCUSSION TITLE
EV charger

### DISCUSSION THREAD

**Discussion by flyingrayson** [tuya/tuya-home-assistant#953] [Q&A]
Hello,

I see the device list supported. I was wondering if there is a possibility to support Electric Vehicule chargers. I just bought one, it appears in Tuya Smart App, and I would very much like it to appear on home assistant in order to manage it directly.

Any chance for this in the (near) future ?

**Comment by pyrousg** [comment]
I have the same need, I have two feyree wallboxes that work very well with the tuya app, but I can't control them in HA, let alone see data from them. The only way to run them from HA is to create a tap to run scene in the tuya app, which will be displayed in HA. But it's such a non-standard solution. It would help me a lot to solve this, I can send json if it would help. Otherwise I've tried everything possible, and impossible. Thank you for your help.

**Reply by grogi** [reply]
@pyrousg That is a fantastic workaround :) 

While having full support in HA would be better, a few scenes exposed from the Smart Life App (Off, 8A, 10A, 13A and 16A) are sufficient for me to control the charging speed and time...

**Reply by pyrousg** [reply]
It is only one option, that really work for me. Actually i have automatisation, which work nice with this tuya wallboxes. Do you know, if there is possibility, how to fully integrate my feyree wallboxes, because after that it should be amazing cheap solution.
